go.mod

```diff
@@ -42,9 +50,11 @@ require (
 require (
 	github.com/axiomhq/hyperloglog v0.2.0
 	github.com/bytedance/sonic v1.12.1
+	github.com/charmbracelet/log v0.4.0
```

Comment:
I think there was slog implemented to remove this dependency. Perhaps it is good to adhere to that.

Reply:
Removed.